README.md

```diff
@@ -3,14 +3,71 @@
 </p>
 <br/>
 
+
 ## Alpaca UI
 
 [![CircleCI](https://circleci.com/gh/SnowdogApps/alpaca-ui/tree/develop.svg?style=svg)](https://circleci.com/gh/SnowdogApps/alpaca-ui/tree/develop)
 
+[![npm version](https://badge.fury.io/js/%40snowdog%2Falpaca-ui.svg)](https://www.npmjs.com/package/@snowdog/alpaca-ui)
+
 Highly customizable library of responsive UI components crafted for e-commerce. It is based on Vue.js and Alpaca design system.
 
 > ⚠️ This library is currently in an early stage. Expect things to be broken or APIs to change as this project matures. DO NOT USE IT IN PRODUCTION!
 
+## Getting Started
+
+### Installation 
+
+Install Alpaca-UI using yarn
```

Comment:
I think it will be better to change it to something like `Add Alpaca UI to your project by using Yarn`